src/main/scala/io/iohk/ethereum/blockchain/sync/SyncStateSchedulerActor.scala

```diff
@@ -53,6 +56,11 @@ class SyncStateSchedulerActor(downloader: ActorRef, sync: SyncStateScheduler)
       targetBlock: BigInt,
       syncInitiator: ActorRef
   ): Receive = {
+    case ReportStats =>
+      syncInitiator ! StateSyncStats(
+        currentStats.saved + currentState.memBatch.size,
```

Comment:
again question of granularity, as if restart happen and membatch won't be persisted then user may end up with lesser number of nodes reporter after restart. If someone is paranoic enought it can mean for him that some data was lost  (i kinda already see QA reporting it that some nodes were lost while testing restarting :D )

Reply:
I see your point. Let me test that as I'm not 100% sure whether that's the wanted behavior